### PR TITLE
Update description in `package.json`

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "JXT_v1.0",
   "version": "1.0.1",
-  "description": "",
+  "description": "Job Search & Tracked Management Version 1.0",
```
Comment (ngl4) [inline package.json]: The description can be renamed "Job Track Management v1.0"

**Review by ngl4 — CHANGES_REQUESTED** [review]
Thanks for letting me know!!

File: package.json (hunk 1 above)
Comment (julieg18) [reply package.json]: Good idea!

**Review by ngl4 — APPROVED** [review]
Looks good!! Thank you!!

**Pull request merged**